viam-orb-slam3.go

```diff
@@ -34,8 +35,6 @@ import (
 	goutils "go.viam.com/utils"
 	"go.viam.com/utils/pexec"
 	"golang.org/x/exp/slices"
-
-	"github.com/viamrobotics/viam-orb-slam3/internal/grpchelper"
```

Comment:
removing this to match behavior of viam-cartographer and keeping all server/client code in RDK